buildlib/io_demo/az-stage-io-demo.yaml

```diff
@@ -0,0 +1,77 @@
+parameters:
+- name: name  # defaults for any parameters that aren't specified
+  default: 'test'
+- name: iodemo_args
+  default: ''
+
+steps:
+- bash: |
+    set -eEx
+    source ./buildlib/az-helpers.sh
+    ./buildlib/az-network-corrupter.sh \
+      initial_delay=$(initial_delay) \
+      cycles=$(cycles) \
+      downtime=$(downtime) \
+      uptime=$(uptime) \
+      |& add_timestamp &>corrupter.log &
+    pgrep -u "$USER" -f 'network-corrupter'
+    corrupter_pid=$(pgrep -u "$USER" -f 'network-corrupter')
+    echo "corrupter_pid=$corrupter_pid"
+    azure_set_variable "corrupter_pid" "$corrupter_pid"
+  displayName: Start network corrupter
+  timeoutInMinutes: 5
+
+- bash: |
+    set -eEx
+    sudo /hpc/local/bin/lshca
+    mkdir -p $(workspace)/${{ parameters.name }}
+    # set UCX environment variables
+    export UCX_SOCKADDR_CM_ENABLE=y
+    #export UCX_NET_DEVICES=$(ibdev2netdev | sed -ne 's/\(\w*\) port \([0-9]\) ==> '${roce_iface}' .*/\1:\2/p')
+    $(workspace)/../test/apps/iodemo/run_io_demo.sh \
+        -H $(agent_hosts) \
+        --tasks-per-node 1 \
+        --duration 60 \
+        -v \
+        --num-clients 1 \
+        --num-servers 1 \
+        --map-by slot \
+        --log-dir $(workspace)/${{ parameters.name }} \
+        -i $(roce_iface) \
+        $(workspace)/../test/apps/iodemo/io_demo \
+            -d 512:524288 \
+            -o read,write \
+            -i 0 \
+            -w 16 \
+            -t 10 \
+            ${{ parameters.iodemo_args }}
+
+  displayName: Launch with run_io_demo.sh ( ${{ parameters.name }} )
+  timeoutInMinutes: 2
```

Comment:
let's increase to 10min

Reply:
@yosefe 
it's timeout for azure. do I increase time for run_io_demo

Reply:
for run_io_demo.sh:
```
	echo "  -d|--duration <seconds>     How much time to run the application"$(show_default_value duration)
```